### DISCUSSION TITLE
Git-Sim Logo

### DISCUSSION THREAD

**Discussion by initialcommit-io** [initialcommit-com/git-sim#65] [General]
@abhijitnathwani @paketb0te Since you two are the first significant contributors to the project, I'd like to get your input to choose a cool Logo for Git-Sim.

Like most of us, I am definitely not a designer but I have 2 options for a logo I've been toying with.  Attaching here (feel free to laugh).

Option 1: Created with Manim in a similar style to Git-Sim output.  We could also create an animated GIF version.

![git-sim-logo_03-04-23_00-33-00](https://user-images.githubusercontent.com/49353917/222916245-ba6edb6d-7edd-4ee8-aa1f-73271040664b.jpg)

Option 2: Random idea that I worked with someone on Upwork to create

![Git-Sim logo 2](https://user-images.githubusercontent.com/49353917/222916381-e0bf6b11-b65c-476a-8b44-e594b38c3adb.jpeg)

Let me know your thoughts and if you have any suggestions.

**Comment by abhijitnathwani** [comment]
This is cool! I like option 1. It resonates well with the overall operations of `git-sim` and stays true to the design sense. All our images and commands more or less output in the same. I'll tinker something on other ways of representing this. In my opinion, we should stick to using Manim to generate the logo. Option 2 looks like a forced spinoff of the original Git logo.

**Comment by initialcommit-io** [comment]
@abhijitnathwani Sounds good I agree!  If you want to play with it I just pushed a new branch called "logo" (https://github.com/initialcommit-com/git-sim/tree/logo) which includes a new command `git-sim logo` which just generates the logo image file. The code for it is in `logo.py` file.

**Comment by abhijitnathwani** [comment]
![G](https://user-images.githubusercontent.com/5513984/222955390-82672386-f37d-4583-80ae-ebbedd84c1e2.png)

@initialcommit-io How about something like this created with Manim in our design schema? The idea is to represent git-sim as a git graph with something branching and merging. I quickly drew this in Canva just to help visualize. I'm not familiar with Manim elements and will take to understand and create with it. If this makes sense, can you try and create something like this with Manim and see how it looks ?

**Reply by initialcommit-io** [reply]
@abhijitnathwani Thx for the suggestion!  Sure I'll try and put something like that together.

**Comment by initialcommit-io** [comment]
@abhijitnathwani @paketb0te Here are some other sample options (sorry there are a bunch), let me know your thoughts. One thing I like about the variations of my original suggestion is we can use the left part as an iconified logo, and the whole thing if we want a long thing that includes the tagline.

![git-sim-logo_03-07-23_16-06-10](https://user-images.githubusercontent.com/49353917/223587890-7926c544-9ce4-4e49-9d85-242f43991b10.jpg)
![git-sim-logo_03-07-23_16-04-32](https://user-images.githubusercontent.com/49353917/223587892-e1702c01-8fc3-4393-9582-fdb9a7db664d.jpg)
![git-sim-logo_03-07-23_16-04-13](https://user-images.githubusercontent.com/49353917/223587893-dc364cd8-7a57-4d4f-b57c-4d7e4dbf8ea3.jpg)
![git-sim-logo_03-07-23_16-02-38](https://user-images.githubusercontent.com/49353917/223587895-305e3e96-6716-4b60-bea5-766c4c618ddb.jpg)
![git-sim-logo_03-07-23_16-00-41](https://user-images.githubusercontent.com/49353917/223587897-33bfc8e8-700e-462b-ac95-a6909e8bc8c5.jpg)
![git-sim-logo_03-07-23_15-58-46](https://user-images.githubusercontent.com/49353917/223587898-9a4e0934-9d3d-4e40-af3c-65f50919bf34.jpg)
![git-sim-logo_03-07-23_15-38-38](https://user-images.githubusercontent.com/49353917/223587900-7d791a33-c09a-4392-bd80-3469802cd510.jpg)
![git-sim-logo_03-07-23_16-27-12](https://user-images.githubusercontent.com/49353917/223587875-4947e9ec-2299-46e6-acdf-043d08cf85cc.jpg)
![git-sim-logo_03-07-23_16-22-32](https://user-images.githubusercontent.com/49353917/223587878-053f26a1-ba3f-4f04-bd48-3d9c7daa8b75.jpg)
![git-sim-logo_03-07-23_16-13-21](https://user-images.githubusercontent.com/49353917/223587881-cc02da89-a125-4ef5-884e-32b257738c64.jpg)
![git-sim-logo_03-07-23_16-11-30](https://user-images.githubusercontent.com/49353917/223587882-158985a2-c15c-4eb9-b7e0-22f1c951ecce.jpg)
![git-sim-logo_03-07-23_16-10-43](https://user-images.githubusercontent.com/49353917/223587886-170d7c5d-a925-4b0c-a591-c09296a08a59.jpg)
![git-sim-logo_03-07-23_16-07-19](https://user-images.githubusercontent.com/49353917/223587889-52d96c00-5d1e-4fd7-be0b-d4cf9b3931c7.jpg)

**Comment by paketb0te** [comment]
Have to admit I prefer the initial option 2 (even though I don't really understand what that thing below the git logo is) , all the other iterations feel a bit "overloaded" to me - there is _so much going on_ :D

I think we should strive to make it more concise, probably with as little text as possible.

I'll ask a designer friend of mine if he would be willing to work with us.

**Reply by initialcommit-io** [reply]
@paketb0te Haha interesting - you're the first person to prefer that one.  My preferences is this one:

https://user-images.githubusercontent.com/49353917/223587875-4947e9ec-2299-46e6-acdf-043d08cf85cc.jpg

The reason is it represents similar output to what Git-Sim actually does, and we can easily iconify it by slicing off the commit chain and just keeping the "circle with 3 commits" around "git-sim" on the left.

But sure if you have a friend that is willing to help us try some variations that would be great.

**Comment by initialcommit-io** [comment]
@paketb0te @abhijitnathwani I'm doing a Git-Sim 3 month dev update tomorrow and so I added an animated logo/tagline to the top of the readme.  Let me know what you think!

I'll be giving you guys shoutouts in the release post as well!  Thanks for the help on this project and looking forward to keep working with you both 😸

**Reply by initialcommit-io** [reply]
(I'll be linking to your github profiles from the release post, and can @ you on Twitter as well, but let me know if there's any other resources of yours that you'd like me to link to)

**Reply by abhijitnathwani** [reply]
The animation looks cool!
Congratulations on the milestone! Gitsim has come a long way since it started 🙌🏻
